akka-docs/src/test/scala/docs/ddata/DistributedDataDocSpec.scala

```diff
@@ -357,9 +358,11 @@ class DistributedDataDocSpec extends AkkaSpec(DistributedDataDocSpec.config) {
   "demonstrate LWWRegister" in {
     def println(o: Any): Unit = ()
     //#lwwregister
-    implicit val node = Cluster(system)
-    val r1 = LWWRegister("Hello")
-    val r2 = r1.withValue("Hi")
+    implicit val node = DistributedData(system).selfUniqueAddress
+    implicit val clock = LWWRegister.defaultClock[String]
```

Comment:
I don't think we should require this, defaultClock should be used by default.

Reply:
Agreed, but it was needed to avoid overloading or default argument duplicate errors - compiler constraints. I'll take another look with the above change.